### DISCUSSION TITLE
星巴克為什麼這麼貴

### DISCUSSION THREAD

**Discussion by 1104030360** [1101-GS4719-C/Python-Community-GS4719-C#1373] [General]
常常有人會認為星巴克所販賣的咖啡價格是外面一般咖啡廳的兩倍，非常貴，但我卻不這麼認為，因為我認為星巴克販賣的不只是咖啡本身還有他所提供的環境，如果大家只是外帶一杯星巴克的咖啡當然會覺得星巴克的咖啡貴，但是當你選擇內用並打開電腦完成代辦工作時就會發現這錢花的物超所值，我認為這也就是為什麼星巴克就算賣貴也不會倒閉的原因之一！

**Comment by TW-Maverick** [comment]
我想要反駁你的看法，星巴克就是那麼的貴！路易莎可以提供一樣的環境與份圍，但價格卻是星巴克的一半，我還是喜歡本土品牌，而且飲品不輸進口。人的價值觀雖然會隨著時間改變，但我目前還是認為星巴克很貴。

**Reply by JessyCheng2003** [reply]
我覺得星巴克的價錢能過濾掉一些素質比較低的客人，在路易莎讀書的氛圍與環境，幾乎都比較吵雜，而且我目前還沒有找到路易莎好喝的飲料🤣。

**Comment by ziihui** [comment]
其實我覺得星巴克不算貴，在外面很多咖啡廳的飲料價錢都比星巴克還高(還不怎麼好喝?，甚至還也有低消的限制~
星巴克很多時候還有買一送一的優惠><在這種優惠期間買就覺得很划算咧

**Reply by quan0715** [reply]
我也覺得星巴克不算貴，而且他更多的是販售品牌（跟蘋果一樣），順帶一提福吉茶拿鐵真的超推！！！

**Comment by Alexgns** [comment]
出國的時候反而覺得星巴克很便宜，手搖飲料反而要更高的價格，所以幾乎每天都喝星巴克當飲料

**Comment by Chiuhsianglin** [comment]
以往我只有在星巴克有買一送一的時候才會去買，現在星冰樂不在買一送一的範圍內了，也讓我買星巴克的機會變得很少。但現在物價越來越高的情況下，感覺星巴克好像也沒有那麼貴了

**Comment by sjh501** [comment]
我個人是覺得星巴克很貴，因為我喝咖啡是為了讓自己更有精神，也知道自己味蕾不敏銳，喝不太出差別，所以以我的需求來說星巴克相比其他咖啡高出很多。但就像有些人提到的環境和附加價值，星巴克所販售的除了咖啡還有它品牌的附加價值，就像是無印良品、蘋果，或者很多看起來就像在課智商稅的東西；除了它的成本和構想，還有本身的品牌和整體的周邊，我認為那些是很難用金錢衡量，也因人而異。

**Comment by zephyrxin01** [comment]
我覺得星巴克和其他有低消和時間限制的咖啡廳比起來真的不算貴耶。我覺得星巴克的環境、質感和整個理念才算是我會選擇星巴克而不是路易莎的原因，而且我覺得星巴克的飲品選擇比較多，有些茶類也很好喝。

**Comment by pigeon9975** [comment]
星巴克雖然貴，但他常常有買一送一的優惠，所以我都趁著買一送一的時間找朋友去享受一波。
同樣的概念還有漢堡王！
對了，在這邊跟大家推薦一個名叫fodiee_diaryy的ig帳號，他會定期更新很多超商、飲料店、速食店的買一送一，是大學生省錢的好方法！

**Comment by aa103967622** [comment]
星巴克的味道比較穩定，咖啡風味並不會有太大的變化，而且星巴克的座位區通常比路易莎來得寬敞、舒適，也比較不會受到隔壁人的干擾，覺得太貴的話就挑買一送一的時期喝星巴克嘛，用路易莎的價錢享有星巴克的咖啡品質、安靜的環境(路易莎通常都吵得要命...)，豈不美哉。

**Comment by alstinlin** [comment]
跟外面的咖啡廳比其來的話好像也不會差很多，因為現在一般的咖啡廳價格也都在一百多左右，而且我覺得星巴克的環境通常都比較寬敞，也蠻乾淨的，至於外帶的話可能就真的要有買一送一才比較划算了

**Comment by ALAN30359704** [comment]
其實星巴克在美國是很便宜的咖啡，所以在美國大為流行，但是由於台灣人的物價較低，所以我們台灣人會覺得星巴克是一個很貴的咖啡。

**Comment by Miinteaa** [comment]
據我觀察星巴克的價格範圍跟外面非連鎖的咖啡廳其實差不多耶！星巴克的產品定價其實也包含了其品牌價值，再加上市場定位不同，其實蠻容易理解為何星巴克能夠在台灣市場佇立這麼久也不會被平價咖啡（尤其是路易莎及超商等等）擊倒的原因～
